docs/book/quick-start.md

```diff
@@ -1007,9 +1003,6 @@ behaviors with the shipped annotations in laminas-form:
   of validator options for the constructor:
   `@Validator({"name": "StringLength", "options": {"min":3, "max": 25}})`.
   This annotation may be specified multiple times.
-- `ContinueIfEmpty`: indicate whether the element can be submitted when it
-  is empty. A boolean value is expected. If `@Required` is set to `false`, this
-  needs to be set to `true` to allow the field to be empty.
```

Comment:
The same here, the feature is deprecated in laminas-inputfilter but at the moment there is no plan for a new major version.